### DISCUSSION TITLE
Can I open a psd file from my file manager in Ps (Open-with>other>Adobe Photoshop)

### DISCUSSION THREAD

**Discussion by albakhtari** [LinSoftWin/Photoshop-CC2022-Linux#27] [Q&A]
Does it work with the new launcher script?

**Comment by MiMillieuh** [comment]
> Does it work with the new launcher script?

It always worked on my DE and distro...

But I might have an idea to fix it

**Reply by albakhtari** [reply]
Yes you're right, it worked on my distro too, but the problem is that if I run use the "open with" Ps starts in that directory. Therefore that is where the cache directory is and my custom workspace settings are not loaded.

**Reply by MiMillieuh** [reply]
> Yes you're right, it worked on my distro too, but the problem is that if I run use the "open with" Ps starts in that directory. Therefore that is where the cache directory is and my custom workspace settings are not loaded.
I'll try to force something I'll see if it fix...

**Reply by albakhtari** [reply]
So it doesn't work?
